### ISSUE TITLE
Install problem

### ISSUE THREAD

**Issue by celson** [voxpupuli/puppet-puppetboard#135]
I tried install puppetboard on CentOS with module puppetboard but when I put this config:
```puppet
  class { 'puppetboard':
    manage_git        => 'lastest',
    manage_virtualenv => 'lastest',
  }
```
This module install python-virtualenv but after that it uninstall python-devel.

I fix it putting this in module:
```puppet
  if $manage_virtualenv and !defined(Package[$::puppetboard::params::virtualenv]) {
    class { '::python':
      virtualenv => 'present',
      **dev => 'present**',
    }
  }
```

**Comment by jyaworski** [comment]
@celson that's our invocation of the python module at https://github.com/stankevich/puppet-python. The behavior of the module must have changed.

**Comment by celson** [comment]
Well, I got your point of view, but the behavior of the module python is package dev is absent if not I set present. If is you that invoke the class you don't need set this to present?

**Comment by jyaworski** [comment]
Honestly, I don't know why the python module ensures absent if it's false. It seems a bit broad to me. That being said, we should probably include the dev parameter in our invocation.

**Comment by celson** [comment]
I think it is a best way. Thanks for consideration.

**Comment by emiljoensson** [comment]
Same issue for me on a fresh master installation. The dev => 'present' fix did the trick for me as well, thanks.

**Comment by Kalki5** [comment]
**Try this.**
https://github.com/skaasguru/puppet-puppetboard

**Comment by kenyon** [comment]
Fixed in #129.
